ocs_ci/ocs/utils.py

```
@@ -2241,3 +2241,46 @@ def get_expected_nb_db_psql_version():
+
+    """
+    storagecluster_obj = OCP(
+        kind=constants.STORAGECLUSTER,
+        namespace=config.ENV_DATA["cluster_namespace"],
+        resource_name=constants.DEFAULT_STORAGE_CLUSTER,
+    )
+    try:
+        resp = storagecluster_obj.get()
+        if resp is None:
+            resp = {}
+    except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)